plugins/outputs/timestream/sample.conf

```
@@ -47,6 +47,26 @@
   ## Specifies if the plugin should create the table, if the table does not exist.
   create_table_if_not_exists = true
 
+  ## Specifies if table should be created with composite partition key
+  ## that determines how your data will be stored and distributed.
+  ## create_table_composite_partition_key is an array of array with type string
```

Does the use of an array of arrays mean a user can define multiple?

```suggestion
  ## create_table_composite_partition_key is an array of arrays with type string
```